doc/user-guide/faq.md

```diff
@@ -136,6 +136,99 @@ You can confirm this by using `ping` or `dig` commands from one of the Exasol no
 
 To update the [DNS settings][exasol-network] for the Exasol cluster, go to ExaSolution &rarr; Configuration &rarr; Network. And update internal DNS server addresses and search domains entries.
 
+## Kerberos Connection Issues
+
+### Kerberos Configuration File Domain Realm
+
+Ensure that the Kerberos, `krb5.conf`, file is correct.
+
+Check that the domain realms are correct after the assignment (`=`) sign. They should be specified in all capital letters. They should not include starting dot (`.`) character after the assignment.
+
+Wrong configuration file example:
+
+```ini
+...
+
+[domain_realm]
+.zone1.example.net = zone1.example.net
+zone1.example.net = zone1.example.net
+.example.net = .EXAMPLE.net
+example.net = EXAMPLE.net
+.exampl-dev.net = .EXAMPLE-DEV.NET
+exampl-dev.net = EXAMPLE-DEV.NET
+```
+
+Correct configuration file example:
+
+```ini
+...
+
+[domain_realm]
+.zone1.example.net = ZONE1.EXAMPLE.NET
+zone1.example.net = ZONE1.EXAMPLE.NET
+.example.net = EXAMPLE.net
+example.net = EXAMPLE.net
+.example-dev.net = EXAMPLE-DEV.NET
+example-dev.net = EXAMPLE-DEV.NET
+```
+
+### Kerberos Configuration File Included Directories
+
+Ensure that Kerberos configuration file, `krb5.conf` does not contain any included directories with additional settings. All Kerberos settings should be available in the `krb5.conf` configuration file.
+
+This can cause problems when using Virtual Schema together with Kerberos connection, because included directories does not exist in the UDF container.
```

Comment:
```suggestion
This can cause problems when using Virtual Schema together with Kerberos connection, because included directories do not exist in the UDF container.
```